service/src/main/java/com/myhome/controllers/HouseController.java

```diff
@@ -16,55 +16,41 @@
 
 package com.myhome.controllers;
 
+import com.myhome.api.HousesApi;
 import com.myhome.controllers.dto.mapper.HouseMemberMapper;
 import com.myhome.controllers.mapper.HouseApiMapper;
-import com.myhome.controllers.request.AddHouseMemberRequest;
-import com.myhome.controllers.response.AddHouseMemberResponse;
-import com.myhome.controllers.response.GetHouseDetailsResponse;
-import com.myhome.controllers.response.ListHouseMembersResponse;
 import com.myhome.domain.CommunityHouse;
```

Comment:
We should use now:
```
import com.myhome.model.CommunityHouse;
import com.myhome.model.HouseMember;
```
rather than:
```
import com.myhome.domain.CommunityHouse;
import com.myhome.domain.HouseMember;
```
because right now we will use generated code for that
So please adjust mappers/services with that change.

Reply:
Domain classes `com.myhome.domain.CommunityHouse` & `com.myhome.domain.HouseMember` contain `OneToMany` & `ManyToOne` annotations which are missing from auto generated classes. It it still requires refactoring, it should go with a new issue.

Reply:
Oh I didn't notice that there are domain classes.
So please change the generated classses names with suffix Dto (CommunityHouse -> CommunityHouseDto, HouseMember -> HouseMemberDto)